app/models/form/lettings/questions/scheme_id.rb

```diff
@@ -9,6 +9,11 @@ def initialize(_id, hsh, page)
     @guidance_position = GuidancePosition::BOTTOM
     @guidance_partial = "scheme_selection"
     @question_number = 9
+    @inferred_answers = {
+      "location.name": {
+        "scheme_has_multiple_locations?": false,
+      },
+    }
```

Comment:
I haven't used this before. I assume it means that if the scheme does not have multiple locations, then we infer the location when the scheme is selected.
I assume this is so that we can clear inferred answers if this is cleared?

Reply:
is location.name used somewhere? As above I haven't used this part of the codebase. I assume it must be, otehrwise why wouldn't the label just be location_name?

Reply:
oh i see. it's the slightly dubiously written `Question:71`